src/Auth0.php

```diff
@@ -613,8 +651,15 @@ public function setIdToken($idToken)
             $sigVerifier = new SymmetricVerifier($this->clientSecret);
         }
 
+        $verifierOptions = [
+            'nonce' => $this->authStore->get('nonce'),
```

Comment:
If the stored values are empty, `null` is returned and the check is skipped

Reply:
I guess this is related to @stevehobbsdev 's question. Devs should not be able to skip this check. As far as I know, the only optional value here is max_age. For leeway you always have a default value of 60 secs, and for nonce you will either get the value set by the dev or one generated by you prior to initialize the auth. 

Please make it required. Happy to discuss internally ⚡️

Reply:
My reply on Steve's comment below covers some of this RE: empty store option. 

The question is ... the nonce is an option in the ID token validator, not a required field. You're saying we should fail here if the nonce, say, was not stored properly and could not be retrieved? So if this is empty, fail?